xarray/tests/test_formatting.py

```diff
@@ -472,6 +474,25 @@ def test_large_array_repr_length():
     assert len(result) < 50
 
 
+@requires_netCDF4
+def test_repr_file_collapsed(tmp_path):
+    arr = xr.DataArray(np.arange(100000), dims="test")
```

Comment:
it seems the default dtype on windows is `int32`, not `int64`, so we'll have to change that:
```suggestion
    arr = xr.DataArray(np.arange(100000, dtype=np.int64), dims="test")
```